src/RemoteFileAPI/Remote.ts

```diff
@@ -52,10 +68,21 @@ export class Remote {
       if (event.code === eventCodeWhenIntentionallyStoppingConnection) {
         return;
       }
-      SnackbarEvents.emit(`Remote API connection closed. Code: ${event.code}.`, ToastVariant.WARNING, 2000);
+
+      //Printing a connetion error alongside the connection closed warning is both redunant and confusing
+      if (successfullyConnected)
+        SnackbarEvents.emit(`Remote API connection closed. Code: ${event.code}.`, ToastVariant.WARNING, 2000);
```

Comment:
```suggestion
      if (successfullyConnected) {
        SnackbarEvents.emit(`Remote API connection closed. Code: ${event.code}.`, ToastVariant.WARNING, 2000);
      }
```
It's not in our lints, but we use curlies even for one-line if statements due to the well-known potential for bugs they can cause otherwise. The sometimes-exception is for `break` and `return`, but we usually write them all on one line in that case.